end_to_end/test_decode_v4_8_one_slice.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+NUM_TOKEN_THRESHOLD=${1}
+OUTPUT_PATH=${2}
```

Comment:
We may need DATASET_PATH as well since base.yml has a default value for it

Reply:
Good catch! I think decode.py doesn't use the training examples - but it does rely on the vocab/tokenizer so we do need to pass in the dataset path.